cou/utils/__init__.py

```diff
@@ -14,8 +14,7 @@
 
 """Utilities for charmed-openstack-upgrader."""
 
-
 import os
 from pathlib import Path
 
-COU_DATA = Path(os.getenv("COU_DATA", "."))
+COU_DATA = Path(f"/home/{os.getenv('USER')}/.local/share/cou")
```

Comment:
Will this work fine with snap and sanp environment variables?

Reply:
Yes it's working as expected. I've manually tested and reading the documentation I didn't find information like the `$HOME` that is override.